tools/sgen/input/FluentUIStyle.yml

```diff
@@ -296,3 +303,10 @@ AP_MSFDrawerTokens:
   shadow2Blur: $Shadow.shadow28.blur2
   shadow2OffsetX: $Shadow.shadow28.x2
   shadow2OffsetY: $Shadow.shadow28.y2
+  minMargin: [horizontal: 44pt, vertical: 20pt]
+  cornerRadius: 14pt
+  dropShadowRadius: 4pt
+  dropShadowOffset: 2pt
+  dropShadowOpacity: 0.05f
```

Comment:
Shouldn't this be mapped to an opacity token?

Reply:
Please see the comment of on mapping here  https://github.com/microsoft/fluentui-apple/pull/476#discussion_r603709325.